assets/setup-wizard/ready/index.js

```diff
@@ -7,6 +7,7 @@ import { MailingListSignupForm } from './mailinglist-signup-form';
 import { formatString } from '../../shared/helpers/format-string.js';
 import { logLink } from '../log-event';
 import { useSetupWizardStep } from '../data/use-setup-wizard-step';
+import useSampleCourse from './use-sample-course';
```

Comment:
I think the name of this might be a bit clearer as something like `useInstallSampleCourse` or `useSampleCourseInstaller`.

Reply:
Good idea! https://github.com/Automattic/sensei/pull/3507/commits/bf79a53126c8e7d73b14c05a02122b3cb25ab2a5